starknet_py/net/schemas/common.py

```diff
@@ -106,6 +95,8 @@ def _deserialize(
 
 class TransactionTypeField(fields.Field):
     def _serialize(self, value: Any, attr: str, obj: Any, **kwargs):
+        if value == TransactionType.INVOKE:
+            return "INVOKE_FUNCTION"
         return value.name if value is not None else ""
```

Comment:
Maybe let's change it to raise if `value.name is None`? I think using default values here wasn't a good idea.

Reply:
Changed to just
```
return value.name
```
It's a serialization function, that uses field already present in some data structure. It shouldn't ever be None.